.cspell.json

```diff
@@ -46,8 +49,6 @@
     "Vdat",
     "XT",
     "XTM",
-    "piyushk",
-    "Piyush",
-    "fout"
+    "yukkysaito"
```

Comment:
What is a git handle doing in here?

Reply:
Is it better to //cspell:ignore the line where it is in the code?
pre-commit requires specifying a user behind every TODO/FIXME so that's why it is in the code

Reply:
I see, yes I prefer cspell:ignore for these as they are (hopefully) termporal.

Reply:
I see there are names all through the cspell though, so let's also look into this at another time as part of a clean-up.